bucket/openjdk.json

```diff
@@ -1,11 +1,11 @@
 {
     "homepage": "https://adoptopenjdk.net/",
-    "version": "9+181",
+    "version": "9.181",
```

Comment:
Should this not be a dash instead of a '.', eg `9-181`? I think it will be easier to read when 9.1.3+22.

Reply:
It should be a ‘+’ as specified by the JEP and my original commit. Build number is separated by a plus and on major version bumps all minor numbers are removed. So 9.0.0+181 => 9+181. It’s unnecessarily complicated imo.

Reply:
Even the oracle java download link labels it as `9+181`: `http://download.oracle.com/otn-pub/java/jdk/9+181/jdk-9_windows-x64_bin.exe`

Reply:
Maybe I misunderstood, but I thought scoop had some issues with + as a version number divider, and therefor @r15ch13 suggested `.` instead of `+`. I just wanted to sync the scoop-extras PR with this one, so that both uses the same version numbers. If that is `+` or `-` I leave up to you guys to decide. I’ll update my PR in extras based on the decision.

Reply:
That's correct. Scoop doesn't accept `+` in the version string. I prefer the hyphen because it's the build number. It makes more sense because otherwise 9.181 > 9.0.1.xxx